mkdocs.yml

```diff
@@ -211,7 +211,7 @@ theme:
       r: material/circle-double
       placeholder: material/pencil-plus
       update-pending: material/information-outline
-      deprecated: material/trash-can
+      deprecated: material/file-remove-outline
```

Comment:
This icon somehow looks like there's a broken file, as if there were an error in our docs. What about this one?

<img width="218" alt="image" src="https://github.com/user-attachments/assets/5b518b76-95c9-411e-94e8-794515100a33" />

```suggestion
      deprecated: octicons/circle-slash-16
```